.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "tools/js-sdk-release-tools/deps/azure-js-dev-tools"]
```

Comment:
Why are we using a submodule? Submodules cause all sorts of issues and we generally avoid them. If you need this dependency then we should publish the package instead of using a submodule.

Reply:
One example of such a issue https://github.com/Azure/azure-sdk-tools/security/dependabot/178/update-logs/370202889

Reply:
Thanks @weshaggard for checking it and kindly share the guidance.

Previous tool owner used a package https://www.npmjs.com/package/@ts-common/azure-js-dev-tools
but looks like no one maintain it any more. The latest package is used, there's no way to easily upgrade the package to resolve the problem. So I add submodule to update the braces package. We will definitely avoid to use packages out of maintainence in the future.

No worries, I'll try shrinkwrap first. If it doesn't help, I'll publish a new package instead of using a submodule.